src/core_atmosphere/physics/mpas_atmphys_manager.F

```diff
@@ -548,10 +549,10 @@ subroutine physics_run_init(configs,mesh,state,clock,stream_manager)
 
 !set alarm for updating the background surface albedo and the greeness fraction:
  call mpas_set_timeInterval(alarmTimeStep,timeString=config_greeness_update,ierr=ierr)
- alarmStartTime = startTime
+    alarmStartTime = startTime
```

Comment:
Checking to see if these added spaces are needed.

Reply:
@AndersJensen-NOAA Line 551 is restored.